docs/pages/docs/styling-theming/themes.mdx

```diff
@@ -54,19 +54,21 @@ Here are each of the theme CSS variables you can set, with values from the defau
 --bn-colors-highlights-pink-text: #ad1a72;
 --bn-colors-highlights-pink-background: #f4dfeb;
 
---bn-font-family: "Inter", "SF Pro Display", -apple-system, BlinkMacSystemFont, "Open Sans", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+--bn-font-family: "Inter", "SF Pro Display", -apple-system, BlinkMacSystemFont, "Open Sans",
+  "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
+  "Helvetica Neue", sans-serif;
 --bn-border-radius: 6px;
 ```
 
-Setting these variables on the `.bn-container[data-color-scheme]` selector will overwrite them for both default light & dark themes. To overwrite variables separately for light & dark themes, use the `.bn-container[data-color-scheme="light"]` & `.bn-container[data-color-scheme="dark"]` selectors.
+Setting these variables on the `.bn-container[data-color-scheme]` selector will overwrite them for both default light & dark themes. To overwrite variables separately for light & dark themes, use the `.bn-container[data-color-scheme="light"]` and `.bn-container[data-color-scheme="dark"]` selectors.
 
-In the demo below, we set a red theme for the editor which changes based on if light or dark mode is selected:
+In the demo below, we set a red theme for the editor which changes based on if light or dark mode is selected (change the website to dark mode in the footer to see the dark theme):
```

Comment:
Should probably be "see this in action by changing the website theme in the footer" since some people will already have dark mode by default